modules/core/src/v2/coins/abstractEthLikeCoin.ts

```diff
@@ -60,6 +62,24 @@ export interface ExplainTransactionOptions {
   feeInfo: TransactionFee;
 }
 
+export interface FullySignedEthLikeTransaction {
```

Comment:
This has the same definition as `FullySignedTransaction`, please import that instead

Reply:
> Can we add `rsk` to the [eth-like coins tests](https://github.com/BitGo/BitGoJS/blob/master/modules/core/test/v2/unit/coins/abstractEthCoin.ts#L16)?
> Then when the [tx is signed](https://github.com/BitGo/BitGoJS/blob/master/modules/core/test/v2/unit/coins/abstractEthCoin.ts#L303), you could verify that the fields exist in the response

RSK is already in eth-like coins tests I believe? (RSK's ticker == `rbtc`)